back-end/schemas/schemas.sql

```diff
@@ -29,13 +29,16 @@ CREATE TABLE IF NOT EXISTS Role (
 -- -----------------------------------------------------
 CREATE TABLE IF NOT EXISTS User (
     user_id INT UNIQUE NOT NULL AUTO_INCREMENT,
-    username VARCHAR(255) NOT NULL,
+    username VARCHAR(255) UNIQUE NOT NULL,
     email VARCHAR(255) UNIQUE NOT NULL,
     password VARCHAR(60) NOT NULL,
     phone VARCHAR(255) UNIQUE NOT NULL,
     role_id INT NOT NULL,
-    PRIMARY KEY (user_id),
+    create_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,
+    updata_time TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
there is a typo, it says `updata` instead of `update`, also you could rename it to `updated_at`, and it is fine if it didn't have a default value.